lib/spotlight/catalog.rb

```diff
@@ -11,17 +11,18 @@ module Catalog
     include Spotlight::Catalog::AccessControlsEnforcement
 
     included do
-      before_filter do
-        if current_exhibit && can?(:curate, current_exhibit)
-          blacklight_config.add_facet_field 'exhibit_visibility',
-                                            label: I18n.t(:'spotlight.catalog.facets.exhibit_visibility.label'),
-                                            query: {
-                                              private: {
-                                                label: I18n.t(:'spotlight.catalog.facets.exhibit_visibility.private'),
-                                                fq: "#{Spotlight::SolrDocument.visibility_field(current_exhibit)}:false" }
-                                            }
-        end
-      end
+      before_action :facet_visibility
+    end
+
+    def facet_visibility
```

Comment:
Could we use a more descriptive name for this filter, maybe: `add_facet_visibility_field`, and add a little method documentation?